### PR TITLE
[DC-1930] Fully convert deactivation data script to cleaning rule

### PR DESCRIPTION
**Pull request by nishanthpp93**
* [DC-1930] Update deactivation CR

[DC-1930]: https://precisionmedicineinitiative.atlassian.net/browse/DC-1930?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by hiro-mishima — CHANGES_REQUESTED** [review]
Thanks for refactoring the scripts, @nishanthpp93! I added a few minor suggestions for changes. Please let me know if you have any questions. Thank you🙂

File: data_steward/cdr_cleaner/cleaning_rules/remove_participant_data_past_deactivation_date.py [hunk 1]
```diff
@@ -27,6 +30,75 @@
     'participantId', 'suspensionStatus', 'suspensionTime'
 ]
 
+DEACTIVATION_ISSUE_NUMBERS = ['DC686', 'DC1184', 'DC1799']
```
[the remark this answers is not in the document]
Comment (nishanthpp93) [reply data_steward/cdr_cleaner/cleaning_rules/remove_participant_data_past_deactivation_date.py:33]: Yeah, I think they are helpful to keep but don't want the names to get too long.

File: data_steward/cdr_cleaner/cleaning_rules/remove_participant_data_past_deactivation_date.py [hunk 2]
```diff
@@ -27,6 +30,75 @@
...
+WHERE ({{datetime}} IS NOT NULL AND {{datetime}} >= d.deactivated_datetime)
+OR ({{datetime}} IS NULL AND {{date}} >= DATE(d.deactivated_datetime))
+{% endif %})
+""")
+
+# Queries to truncate existing tables to remove deactivated EHR PIDS, two different queries for
+# tables with standard entry dates vs. tables with start and end dates
+CLEAN_QUERY = JINJA_ENV.from_string(  # language=JINJA2
+    """
+SELECT *
```
[the remark this answers is not in the document]
Comment (nishanthpp93) [reply data_steward/cdr_cleaner/cleaning_rules/remove_participant_data_past_deactivation_date.py:85]: Good point, will update this.

[the remark this answers is not in the document]
Comment (nishanthpp93) [reply data_steward/cdr_cleaner/cleaning_rules/remove_participant_data_past_deactivation_date.py:85]: This seems harder to implement due to fitbit tables not having an id column. Currently this is schema agnostic. Let me know your thoughts.

[the remark this answers is not in the document]
Comment (hiro-mishima) [reply data_steward/cdr_cleaner/cleaning_rules/remove_participant_data_past_deactivation_date.py:88]: You are right. I think this is out of this ticket's scope. Let's leave it as it is.

**Review by hiro-mishima — APPROVED** [review]
Thanks, @nishanthpp93! It all looks good!

**Pull request merged**